src/Backend/Modules/Quotes/info.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module>
   <name>Quotes</name>
-  <version>1.0.0</version>
+  <version>2.0.0</version>
   <requirements>
-    <minimum_version>4.0.0</minimum_version>
+    <minimum_version>5.0.6</minimum_version>
```

Comment:
technically this is just 5.0.0

Reply:
technically, the version i'm creating this on is 5.0.6. Who knows what sort of crazy bugfixes I'm relying on?